ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -299,6 +297,16 @@ class KtlintCommandLine {
         }
     }
 
+    private fun configureLogging() {
+        loggerModifier = { kLogger ->
+            (kLogger.underlyingLogger as Logger).level = when {
+                trace -> Level.TRACE
```

Comment:
If there need a cast check like this?
```kotlin
(kLogger.underlyingLogger as? Logger)?.level
```

Reply:
A more strict cast will allow "fail fast" behaviour and will be instantly visible if anyone would try to change underlying logger implementation for `ktlint` module. Otherwise, it can lead to an annoying problem where passing parameter just does not do anything since casting error is silently swallowed.